.rubocop.yml

```diff
@@ -7,7 +7,7 @@ AllCops:
 
 # Baseline code complexity metrics - we should try and reduce these over time
 Metrics/AbcSize:
-  Max: 47
+  Max: 71.5
```

Comment:
Ugh. I'd rather have this cop disabled only in the exceptions (affected files) rather than across the board.